usaspending_api/common/data_classes.py

```diff
@@ -10,7 +10,18 @@ class Pagination:
     upper_limit: int
     sort_key: Optional[str] = None
     sort_order: Optional[str] = None
+    secondary_sort_key: Optional[str] = None
+
+    @property
+    def _sort_order(self):
```

Comment:
I would advise against a function with the same name as a variable. In this case it might be easiest just to leave the logic within order_by()

Reply:
It's not the same name

Reply:
A future reader will mistake the two for each other.

Reply:
That will teach them for using a property with a naming signal to denote internal use only!